sources/OpenMcdf.Extensions/OLEProperties/PropertyFactory.cs

```diff
@@ -1,35 +1,20 @@
 ﻿using OpenMcdf.Extensions.OLEProperties.Interfaces;
 using System;
-using System.Collections.Generic;
 using System.Text;
 using System.IO;
-using System.Threading;
 
 namespace OpenMcdf.Extensions.OLEProperties
 {
-    internal class PropertyFactory
+    internal abstract class PropertyFactory
     {
-        private static ThreadLocal<PropertyFactory> instance

```

Comment:
As it stands, the factory class is stateless, so I don't know why it needs to be ThreadLocal? (if not, that change could be broken out into a separate PR to make things simpler here)